### PR TITLE
init commit for Routes

### PR DESCRIPTION
**Pull request by kaeline**
https://github.com/Metatavu/vp-kuljetus-work-planning-service-api/issues/2

### CODE REVIEW

File: src/main/resources/db/changeLog.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+    <changeSet id="routes" author="katja.danilova">
+        <createTable tableName="Route">
```
Comment (villejuutila) [inline src/main/resources/db/changeLog.xml:8]: Use lowercase table names in migrations. And also all lowercase column names.

File: src/main/kotlin/fi/metatavu/vp/persistence/AbstractRepository.kt [hunk 2]
```diff
@@ -0,0 +1,130 @@
+package fi.metatavu.vp.persistence
```
Comment (villejuutila) [inline src/main/kotlin/fi/metatavu/vp/persistence/AbstractRepository.kt:1]: Shouldn't the package name be `fi.metatavu.vp.workplanning`?
Follows the convention within project and also keeps it clearer once the micro services start communicating together.

**Review by villejuutila — APPROVED** [review]

File: src/main/kotlin/fi/metatavu/vp/persistence/AbstractRepository.kt (hunk 2 above)
Comment (kaeline) [reply src/main/kotlin/fi/metatavu/vp/persistence/AbstractRepository.kt:1]: It should, i forgot about the package name.

File: src/main/resources/db/changeLog.xml (hunk 1 above)
Comment (kaeline) [reply src/main/resources/db/changeLog.xml:8]: Why always change the default Hibernate logic of using capitalized table names? 
@anttileppa what do you think?

**Pull request merged**